doc/configuration.rst

```diff
@@ -186,6 +184,46 @@ As the patterns are parsed as `regular expressions`_, users are advised to consu
 
         $ sphinx-build -D sphinx_gallery_conf.filename_pattern=plot_specific_example\.py ...
 
+.. _run_stale_examples:
+
+Rerunning stale examples
+========================
+By default, sphinx-gallery only rebuilds examples that have changed.
+For example, starting from a clean ``doc/`` directory, running your HTML build
+once will result in Sphinx-gallery executing all examples that match your given
+filename / ignore patterns. Then, running
+the exact same command a second time *should not run any examples*, because the
+MD5 hash of each example will be checked against the MD5 hash (saved to disk
+as ``<filename>.md5`` in the generated directory) that the example file had
+during the first build, they will match and thus the example will be determined
+to be "stale", and it will not be rebuilt by Sphinx-gallery.
+This design feature allows for more rapid documentation iteration by only
+rebuilding examples when they change.
+
+However, this presents a problem during some modes of debugging and
+iteration. Let's say that you have one particular
+example that you want to rebuild repeatedly while modifying some function in
+your underlying library, while leaving the example file contents themselves
+fixed. To do this, you'd either need to make some change (e.g., add/delete a
+newline) to your example or delete the ``.md5`` file to force Sphinx-gallery
+to rebuild the example. Instead, you can use the configuration value::
+
+    sphinx_gallery_conf = = {
+        ...
+        'run_stale_examples': True,
+    }
+
```

Comment:
With this configuration all examples will be rebuilt, even if their MD5 hash show that the examples have not changed.